packages/phrases/src/locales/en/translation/admin-console/sign-in-exp.ts

```diff
@@ -71,15 +71,15 @@ const sign_in_exp = {
     },
     languages: {
       title: 'LANGUAGES',
-      enable_auto_detect: 'Enable auto detect',
+      enable_auto_detect: 'Enable auto-detect',
       description:
-        "Your software detects the user's location and switches to the local language. You can add new locales by translating UI from English to another language.",
+        "Your software detects the user's location and switches to the local language. You can add new languages by translating UI from English to another language.",
```

Comment:
I think it better to say it detects the "user locale" instead of "user's location". They are totally different and location seems to more private than locale and people who cares about privacy might be concerned.